app/views/docs/_definitions.html.erb

```diff
@@ -0,0 +1,41 @@
+<h2 id="definitions">Definitions</h2>
+
+<% @swagger.definitions.each do |definition| %>
+<h3 id="<%= definition.unique_key %>"><%= definition.name %></h3>
+
+<% if definition.description %>
+<%= definition.html_description %>
+<% end %>
+
+<code>
+  <pre>
+    {
+    <% definition.properties.each do |property| %>
+      <% if property.comment %>
+        /* <%= property.comment %> */
```

Comment:
It's actually not commented out but a trick to insert a comment in the JSON. I should explain that I am playing around with different ways of annotating types